demisto_sdk/commands/validate/validators/BC_validators/BC107_is_valid_toversion_on_modified.py

```diff
@@ -14,30 +16,112 @@
 
 class IsValidToversionOnModifiedValidator(BaseValidator[ContentTypes]):
     error_code = "BC107"
-    description = (
-        "Check that the toversion property was not changed on existing Content files."
-    )
+    description = "Checks that the `toversion` property was not changed on existing content, unless it is replaced by a new file with the same ID and a continuous fromversion."
     rationale = "Changing the `toversion` field for a content item can break backward compatibility."
-    error_message = "Changing the maximal supported version field `toversion` is not allowed. Please undo, or request a force merge."
+    error_message = "Changing the maximal supported version field `toversion` is not allowed. unless you're adding new content_item with the same id {0} and their from/to version fulfills the following:\n{1}"
     related_field = "toversion"
-    expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.RENAMED]
+    expected_git_statuses = [
+        GitStatuses.MODIFIED,
+        GitStatuses.RENAMED,
+        GitStatuses.ADDED,
+    ]
+    valid_cases = (
+        "The old item `toversion` field should be less than the new item `fromversion` field\n"
+        "The old and the new item should be continuous, aka the old one `toversion` is one version"
+        " less than the new one `fromversion`"
+    )
```

Comment:
Is there a reason why we don't add is as a part of the error_message?
I don't think it is necessary to use a separate variable.